### ISSUE TITLE
Derelict MoMMIs Can't Get Precious Metals

### ISSUE THREAD

**Issue by Shadowmech88** [d3athrow/vgstation13#3628]
As it is right now, there's no way for MoMMIs on the Derelict to get access to refined gold, silver, plasma, uranium, diamond, bananium, or phazon. They could easily mine all of those on the Clown Roid, but there is no ore redemption machine on the Derelict, and no circuit board exists to build one. Such materials are useful for enabling the MoMMIs to construct hyper-capacity power cells and upgrade each other, among other things.

I'm not sure what solution would be best, but there needs to be one.

**Comment by CptWad** [comment]
I always thought a refresher on derelict would be cool. Sprucing it up with a lot less random rwalls and structures that were not in the original design. Its extremely difficult to rebuild the derelict as time is always against you. it takes three hours alone just to power, set up atmos, then make most of it air tight.
There is a question of adding smelters/ore redemption to the circut imprinters but then science will be able to mine and process their own materials which is a huge no no.
I think adding some simple smelter on derelict somewhere will work. Problem is that humans will mine out the clown roid for their own reasons.

I think its best to leave it as is. Mommis can synth plas-steel/glass so they never have to worry to the larger engineering degree. Gold/silver/uranium/diamond are the only materials needed for anything in rnd. Mommis only need some three or four items from rnd for their work but I would have to look over to see which materials are really needed. Hyper cells just need silver if I recall but as a derelict mommi I have never had issues with my power supply and I find having huge power cells take FUCKLONG to charge. The power cell dere-mommis start with is the most balanced in this aspect

tl;dr: If there is a solution its not needed immediately, so eh. :neckbeard:

**Comment by Shadowmech88** [comment]
Upgrading a machine to tier 3 requires uranium if it has a micro-laser in its parts.

**Comment by clusterfack** [comment]
Both solutions I've thought of to the solution are just not workable for various reasons.

Obviously you could have the derelict start with some resources for example, that just ends up with telescience teleporting them in every single round within 5 minutes.

Or you could have mommis able to mine the clown roid. Which means now humans and derelict mommis both have to make a mad dash to it to steal all the resources from it before the other one does, and one of the two is going to get screwed over.

I'd like to see some way for it to work out but nothing has come to mind.

**Comment by ComicIronic** [comment]
Giving MoMMIs the ability to hop one Z-level to the roid might be a solution (with the obvious requirement of making sure they can't shuttle away.)
